### DISCUSSION TITLE
gsmartcontrol, hardinfo redshift-gtk

### DISCUSSION THREAD

**Discussion by constantius24** [mocaccinoOS/mocaccino#66] [General] [answered]
A know that is beta of system. But there is lack of some packages. They dont exist in flatpak and similar. For example gsmartcontrol, hardinfo, redshift, leafpad, cairo-clock. Could you add it on luet repository... Or the main qeustion. When your repu will have full packages as Sabayon.

**Comment by joostruis** [comment, marked as answer]
It is a work in progress. I think you best open an issue for package requests in the community repository here:
https://github.com/mocaccinoOS/community-repository/issues

**Comment by joostruis** [comment]
![hwinfo](https://user-images.githubusercontent.com/1949179/163682418-f4071649-e194-4cd6-8988-fa25dfe93545.png)
Available in community repo now.

**Reply by constantius24** [reply]
works - thanks for your job and support
